.husky/pre-commit

```diff
@@ -1,4 +1 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-
```

Comment:
After noticing a deprecation notice after husky upgrade, I've followed the process here https://github.com/typicode/husky/releases/tag/v9.0.1